sycl/include/sycl/detail/pi.h

```diff
@@ -271,6 +273,7 @@ typedef enum {
   PI_DEVICE_INFO_IMAGE_SRGB = 0x10027,
   // Return true if sub-device should do its own program build
   PI_DEVICE_INFO_BUILD_ON_SUBDEVICE = 0x10028,
+  PI_EXT_INTEL_DEVICE_INFO_FREE_MEMORY = 0x10029,
```

Comment:
Other EXT enumerations step the value up, i.e. `PI_EXT_ONEAPI_DEVICE_INFO_*` starts at `0x20000`. I think it would be best to do that with `PI_EXT_INTEL_DEVICE_INFO_*` too. Either as a collective `PI_EXT_*_DEVICE_INFO_*` numbering, or by taking another leap in numbers, for example `0x30000` for `PI_EXT_INTEL_DEVICE_INFO_*`. I think I prefer the latter.

Reply:
Other PI*_DEVICE_INFO* around where I put new one, are already Intel extension (since line 263), they just weren't accurate to use PI_EXT_INTEL as a prefix. Not sure if finding another unused range is meaningful then

Reply:
I see! In that case the current value seems fitting. Thanks for clarifying.